### PR TITLE
Debuff French Curse (fixed string error)

### PR DESCRIPTION
**Pull request by kilkun**
This is a notice to all Wizards,

Attempting to work your dark magic on one of the Honkmother's chosen to warp their pure form into a twisted perverse space french street performer is ill advised.

The Honkmother's children have been blessed with her infinite honking wisdom and will deflect your magical energy directly back into you, multiplied a thousandfold. Do not attempt to apply the wretched frenchman's curse to a Nanotrasen standard-issue clown, or the Space Wizards Federation will have to pull support for your mission immediately.

Regards,
-The Grand Master of the Space Wizards Federation

Changes:
-Clowns are immune to the Frenchman's curse spell
-Attempting to force a clown into mimehood will result in the spell user to gib after 5 ticks.

### CODE REVIEW

File: code/modules/spells/targeted/equip/frenchcurse.dm [hunk 1]
```diff
@@ -25,9 +25,15 @@
...
 	..()
 	for(var/mob/living/carbon/human/target in targets)
-		flick("e_flash", target.flash)
-		target.miming = 1
-		target.add_spell(new /spell/aoe_turf/conjure/forcewall/mime)//They can't even acid the mime mask off, if they're going to be permanently muted they may as well get the benefits of the mime.
+		if(target.mind.assigned_role=="Clown")
+			usr << "<span-class='warning'>Your magic energy is reflected back into you a thousand fold. Uh oh.</span>"
+			target << "<span-class='confirm'>The blessing of the honkmother repels the vile assault!</span>"
+			sleep(5)
+			usr.gib()
```
Comment (PJB3005) [inline code/modules/spells/targeted/equip/frenchcurse.dm]: \>not even using `user`

Also, a straight up gib is insane.

**Comment by Konater** [comment]
:-1:
what would it do to people who are clown cursed though?

**Comment by kilkun** [comment]
Nothing because their assigned_role isn't set to CLOWN

**Comment by Intigracy** [comment]
>clown hides via changing clothes
>you try to french curse the clown in an assistant's clothing
>you gib without warning

yeah nah
:-1:

**Comment by kilkun** [comment]
>clowns not wearing their mask at all times so you can easily identify them
>Wizards not checking crew manifest
they were asking for it tbh

**Comment by PJB3005** [comment]
Gas mask + no ID

**Pull request closed without merging**